projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -279,4 +283,27 @@ public function calculate_scheduled_shares() {
 	public function social_filter_available_modules( $modules ) {
 		return array_merge( array( self::JETPACK_PUBLICIZE_MODULE_SLUG ), $modules );
 	}
+
+	/**
+	 * Set up sharing limits.
+	 */
+	public function set_up_sharing_limits() {
+		global $publicize;
+
+		$info = $publicize->get_publicize_shares_info( \Jetpack_Options::get_option( 'id' ) );
+
+		if ( is_wp_error( $info ) ) {
+			return;
+		}
+
+		if ( empty( $info['is_share_limit_enabled'] ) ) {
+			return;
+		}
+
+		$connections      = $publicize->get_all_connections();
+		$shares_remaining = 1; // @TODO: add this calculated number to the endpoint.
```

Comment:
Just confirming that this is hard-coded just for testing purposes.

Reply:
No, I forgot to push a change 😂 will fix!